web/source/build.sh

```diff
@@ -222,13 +228,13 @@ if [ $BUILD_EMBED = true ]; then
         fail "Typescript compilation failed."
     fi
     assert $INTERMEDIATE/keyman.js
-    echo Embedded TypeScript compiled.
-
-
-    minify keyman.js $EMBED_OUTPUT SIMPLE_OPTIMIZATIONS "KeymanBase.__BUILD__=$BUILD"
-    assert $EMBED_OUTPUT/keyman.js 
+    echo Embedded TypeScript compiled as $INTERMEDIATE/keyman.js
 
-    echo Compiled embedded application saved as $EMBED_OUTPUT/keyman.js
+    if [ $DO_MINIFY = true ]; then
+        minify keyman.js $EMBED_OUTPUT SIMPLE_OPTIMIZATIONS "KeymanBase.__BUILD__=$BUILD"
+        assert $EMBED_OUTPUT/keyman.js
+        echo Compiled embedded application saved as $EMBED_OUTPUT/keyman.js
+    fi
```

Comment:
When I build `./build.sh -no_minify`, I'm not seeing a current `/web/release/embedded/keyman.js`. Shouldn't the build script copy `keyman.js` to `$EMBED_OUTPUT/`? 

And for keymanweb.js?

The `asserts` probably should be done after the `fi` statements